cypress/integration/ui/quickMaps/baseline.ts

```diff
@@ -37,14 +39,16 @@ describe('Quick maps - Baseline', () => {
   /* ==== Test Created with Cypress Studio ==== */
   it('baseline estimate panel showing correct data', function () {
     /* ==== Generated with Cypress Studio ==== */
-    cy.visit('/quick-maps/food-systems/baseline?country-id=MWI&mnd-id=Ca&age-gender-group-id=WRA&sm=0&si=');
+    cy.visit(
+      '/quick-maps/food-systems/baseline?country-id=MWI&mnd-id=Ca&measure=food systems&age-gender-group-id=WRA&sm=0&si=',
+    );
     cy.wait(3000);
-    cy.get('.nation > .estimateValue').should('be.visible');
-    cy.get('.nation > .estimateValue').should('have.text', ' 296 ');
-    cy.get('.vitamin > .estimateValue').should('be.visible');
-    cy.get('.vitamin > .estimateValue').should('have.text', '759');
-    cy.get('.baseline > .estimateValue').should('be.visible');
-    cy.get('.baseline > .estimateValue').should('have.text', ' -61% ');
+    cy.get('.median > .estimateValue').should('be.visible');
+    cy.get('.median > .estimateValue').should('have.text', ' 88.2 ');
+    cy.get('.adequacy > .estimateValue').should('be.visible');
+    cy.get('.adequacy > .estimateValue').should('have.text', '814');
+    cy.get('.prevalence > .wrap-15').should('be.visible');
+    cy.get('.prevalence > .wrap-15').should('have.text', '99.87% of sampled households (12396/12412)');
```

Comment:
These tests are a tad flakey given the potential for data to change, but the edits here should "fix" the current test so that is good.

At some point we should refactor the test to locally call the API and then confirm the presented data matches the API response, but that can be for future Janusz :stuck\_out\_tongue\_closed\_eyes: